### PR TITLE
Chat-Detail-Title-Ui

### PR DESCRIPTION
**Pull request by yashkapoor8710**

### CODE REVIEW

File: config/DefaultConfig.ts [hunk 1]
```diff
@@ -49,6 +50,7 @@ export const defaultConfig: ApplicationConfig = {
         chatHourDisplayFormat: "hh:mm A",
         chatYesterdayText: "Yesterday",
       },
-      searchPlacerHolder: "Search Chats"
+      searchPlacerHolder: "Search Chats",
+      chatDetailTitle: "white"
```
Comment (manojsinghnegiwd) [inline config/DefaultConfig.ts]: Please remove this.

File: pages/ChatDetails/index.tsx [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
-import { Text } from 'react-native';
+import ChatShow from '../../components/Chat/ChatShow';
+import { RouteComponentProps } from 'react-router-native';
+import { Dispatch } from 'redux';
 
-interface Props {};
+interface Props extends RouteComponentProps {
+    dispatch: Dispatch
+}
+
```
Comment (manojsinghnegiwd) [inline pages/ChatDetails/index.tsx]: remove dispatch from props.

File: components/Chat/ChatShow.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+const ChatShow: React.FunctionComponent<Props> = ({
+    onChatPress,
+    userImageSource,
+    userName,
+    status,
+}: Props) => {
+    const theme: AppTheme = useTheme();
+    const constants: AppConstants = useConstants();
+    return (
```
Comment (manojsinghnegiwd) [inline components/Chat/ChatShow.tsx]: Add a newline before return.

File: components/Chat/ChatShow.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+import Icon from 'react-native-vector-icons/Ionicons';
+import MaterialIcon from 'react-native-vector-icons/MaterialCommunityIcons';
+import useConstants from '../../hooks/useConstants';
+
+interface Props {
+    userImageSource: ImageSourcePropType;
+    userName: string;
+    status: string;
+    onChatPress?: (event: GestureResponderEvent) => void
+};
```
Comment (manojsinghnegiwd) [inline components/Chat/ChatShow.tsx]: rename `onChatPress` to `onBackPress`

Comment (yashkapoor8710) [reply components/Chat/ChatShow.tsx]: I have renamed it

File: components/Chat/ChatShow.tsx (hunk 3 above)
Comment (yashkapoor8710) [reply components/Chat/ChatShow.tsx]: New Line Added.

File: config/DefaultConfig.ts (hunk 1 above)
Comment (yashkapoor8710) [reply config/DefaultConfig.ts]: I have removed this

File: pages/ChatDetails/index.tsx [hunk 5]
```diff
@@ -1,16 +1,14 @@
 import React from 'react';
 import { ScrollView } from 'react-native';
-import ChatItem from '../../components/Chat/ChatItem';
-import { Text } from 'react-native';
+import ChatShow from '../../components/Chat/ChatShow';
```
[the remark this answers is not in the document]
Comment (yashkapoor8710) [reply pages/ChatDetails/index.tsx]: I have changed the name

File: pages/ChatDetails/index.tsx (hunk 2 above)
Comment (yashkapoor8710) [reply pages/ChatDetails/index.tsx]: I have removed this

File: pages/ChatDetails/index.tsx [hunk 6]
```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import { ScrollView } from 'react-native';
-import ChatItem from '../../components/Chat/ChatItem';
-import { Text } from 'react-native';
+import ChatDetailHeader from '../../components/Chat/ChatShow';
+import { RouteComponentProps } from 'react-router-native';
```
Comment (manojsinghnegiwd) [inline pages/ChatDetails/index.tsx]: Move this up to 3rd line all the imports from npm packages should be on top.

Comment (yashkapoor8710) [reply pages/ChatDetails/index.tsx]: I have moved it

File: components/Chat/ChatShow.tsx [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+    userNameStyle: TextStyle;
+}
+
+const style: Style = StyleSheet.create<Style>({
+    container: {
+        flexDirection: 'row',
+        padding: 20,
+        borderBottomWidth: 1,
+        borderBottomColor: '#cccccc',
+    },
```
Comment (manojsinghnegiwd) [inline components/Chat/ChatShow.tsx]: Please use `lightBottomColor` from constants.

**Pull request merged**